default/stringtables/en.txt

```diff
@@ -1795,6 +1795,9 @@ Use Binary serialization for saving games. Binary serialization is faster to sav
 OPTIONS_DB_XML_ZLIB_SERIALIZATION
 When saving games with XML serialization, compress most of the XML before writing the file. Compression substantially reduces save file sizes, but may make saves unloadable due to memory requirements to decompress the save data.
 
+OPTIONS_DB_HOSTLESS_SAVE
```

Comment:
should call this something with `AUTOSAVE` in the name, and put it with the other autosave strings.

Reply:
Renamed and moved.